xcode/App-Shared/ViewController.swift

```diff
@@ -27,7 +27,7 @@ class ViewController: UIViewController, WKNavigationDelegate, WKScriptMessageHan
 		logger?.debug("\(#function, privacy: .public) - DEBUG mode: isInspectable = true")
 #endif
 		view = webView
-		self.webView.scrollView.isScrollEnabled = false
+//		self.webView.scrollView.isScrollEnabled = false
```

Comment:
Unsure if it was purposeful to leave this in, if so, you can ignore this comment.